presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -168,6 +172,7 @@ public IcebergHiveMetadata(
         super(typeManager, functionResolution, rowExpressionService, commitTaskCodec, nodeVersion, filterStatsCalculatorService, statisticsFileCache);
         this.metastore = requireNonNull(metastore, "metastore is null");
         this.hdfsEnvironment = requireNonNull(hdfsEnvironment, "hdfsEnvironment is null");
+        this.filterStatsCalculatorService = requireNonNull(filterStatsCalculatorService, "filterStatsCalculatorService is null");
```

Comment:
why is this added? We already have this field in the superclass

Reply:
@ZacBlanco, super doesn't expose any getter , thats why i kept it here. Do you want to expose getter in base class?

Reply:
Why does it need a getter though? The field in the superclass is protected so it can be used in subclasses